### DISCUSSION TITLE
Completely not working pdf viewer

### DISCUSSION THREAD

**Discussion by Miedziaq** [stephanrauh/ngx-extended-pdf-viewer#2618] [Q&A]
Hi, i used this library 3 years ago with angular 11 and it works well without problems, 15minutes to configurate and its it.

For now i want to use it with angular 16, and im stuck completely. 
Every example/docs are completely unglued......

I tried with url, blob, just asset but the pdf viewer is not loaded at all and there is no errors in console...
The only one thing in console is 

> PDF 1cc0a0beb0889d1aaa8132a7f3d84ee7 [1.6 LibreOffice 7.4 / Writer] (PDF.js: 4.7.689 [b1e2dc567])  modified by ngx-extended-pdf-viewer 22.0.0-alpha.2

Finally i tried automatic way "ng add ngx-extended-pdf-viewer@latest" but it is still not working :|

**Comment by stephanrauh** [comment]
Strange. What about Angular 17? Maybe I've broken Angular 16 support without noticing. Another idea: create a fresh new project with `ng new` and add the viewer with `ng add ngx-extended-pdf-viewer`. Does it work now?

**Reply by stephanrauh** [reply]
I just create a new project with Angular 16.2.12, added the PDF viewer with `ng add ngx-extended-pdf-viewer`, and it works. What's different in your project?

**Reply by Miedziaq** [reply]
I really dont know, i reinstalled it again and it works, but its not working on azure server with error
![image](https://github.com/user-attachments/assets/b79e7c91-62df-473b-b9d2-624d1d9cc451)

**Reply by stephanrauh** [reply]
You need to configure your server. For some reason unknown, many servers still don't support *.mjs files by default, so you have to tell the server that this is JavaScript. Find more details here: https://pdfviewer.net/extended-pdf-viewer/troubleshooting

**Reply by Miedziaq** [reply]
yea you are right, looks like missed mjs in mimeconfig :) i will check it

thx
